assets/js/components/App.tsx

```diff
@@ -51,6 +53,10 @@ class AppRoutes extends React.Component {
             </Route>
             <Route path="pa-messages" element={<PaMessagesPage />} />
             <Route path="pa-messages/new" element={<NewPaMessagePage />} />
+            <Route
```

Comment:
I don't think the user should be able to get to this page from anywhere but the link on `pa-messages/new`. Sloane and I thought about the same thing in the Select Stations task and came to [this solution](https://github.com/mbta/screenplay/blob/select-stations-page/assets/js/components/Dashboard/NewPaMessage/NewPaMessage.tsx). Basically, each of these separate pages are an Enum value and this container component conditionally renders based on what value is saved in state. This allows the pages to feel separate without needing to handle the user navigating directly to the page with this specific URL.